timeseries_download.py

```diff
@@ -39,7 +39,7 @@ def choose_downloader(downloaders_dict : dict) -> str:
         choice = input("Choose between: {} ".format(list(downloaders_dict.keys())))
         if(choice in downloaders_dict.keys()):
             break
-        print("Unable to parse ", choice)
+        log.i("Unable to parse ".format(choice))
```

Comment:
Codacy found an issue: [Too many arguments for format string](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5768570)